packages/start-frontend/src/helpers/prompt-clack.ts

```diff
@@ -0,0 +1,193 @@
+          placeholder: "./my-app",
+          initialValue: `./${dir}` || "",
+          validate: (value) => {
+            if (!value) {
+              return "Please provide a location path";
+            }
+            // regular expression that text start from ./
+            const Reg = new RegExp("^\\./");
+            if (!Reg.test(value)) {
+              return "Please input a valid location path";
+            }
+          },
+        }),
+      jsLibrary: () => {
+        return select({
+          message: "Select a JavsScript library for UI",
+          options: Object.keys(CLIOptions).map((key) => {
+            return {
+              value: key,
+              label: CLIOptions[key].name,
+            };
+          }),
+        }) as Promise<string>;
+      },
+
+      // select example
+      apiSolution: () => {
+        return select({
+          message: "Select an API Solution (Use arrow keys)",
+          options: CLIOptions.react.apiSolution.map((solution) => {
+            return {
+              value: solution.value,
+              label: solution.name,
+            };
+          }),
+        }) as Promise<"graphql" | "restful">;
+      },
+
+      modules: (value) => {
+        return multiselect({
+          message: "Select module do you want to use (Press 'space' to select)",
+          options: CLIOptions[value.results.jsLibrary as string].useModules,
+          required: false,
+        }) as Promise<string[]>;
+      },
+
+      isUseSampleTestCode: () => {
+        return confirm({
+          message: "Add Testing codes for Catching bugs early?",
+        });
+      },
+    },
+    {
+      // On Cancel callback that wraps the group
+      // So if the user cancels one of the prompts in the group this function will be called
+      onCancel: ({ results }) => {
+        cancel("Operation cancelled.");
+        process.exit(0);
+      },
+    }
+  );
+
+  const testGroupUtility = await group({
+    useVitest: () => {
+      return confirm({
+        message: "Add Vitest for Unit Testing?",
+      });
+    },
+    useStorybook: () => {
+      return confirm({
+        message: "Add Storybook for Visual Testing?",
+      });
+    },
+    useE2E: () => {
+      return confirm({
+        message: "Add Playwright for End-To-End Testing?",
+      });
+    },
+    useEslint: () => {
+      return confirm({
+        message: "Add ESLint for Code Linting?",
+      });
+    },
+    usePrettier: () => {
+      return confirm({
+        message: "Add Prettier for Code Formatting?",
+      });
+    },
+  });
+
+  // spinner example
+  const s = spinner();
+  s.start("Installing fake api solution with fake location");
+
+  await sleep(3000);
+
+  s.stop(`Installed fake modules successfully`);
+
+  // outro example
+  outro(
+    `Finish setting up! 
+    Here is your configurations:
+    JS library: ${color.blue(groupUtility.jsLibrary)}
+    Options: ${color.blue(
```

Comment:
I think this should be "Modules", instead of "Options", to match the previous prompts.